<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reglament</class>
 <widget class="QDialog" name="reglament">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reglament</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>361</width>
     <height>339</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Турнирный регламент</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>60</y>
      <width>121</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Начало</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>99</width>
       <height>207</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTimeEdit" name="tur1begin">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur2begin">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur3begin">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur4begin">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur5begin">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur6begin">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>120</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Конец</string>
    </property>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>99</width>
       <height>207</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTimeEdit" name="tur1end">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur2end">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur3end">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur4end">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur5end">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="tur6end">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>1-й тур</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>2-й тур</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>3-й тур</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>4-й тур</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>5-й тур</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>6-й тур</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>346</y>
     <width>154</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ПРИМЕНИТЬ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>reglament</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>reglament</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
